app/Repositories/LandingPageRepository.php

```diff
@@ -0,0 +1,205 @@
+            'routePrefix' => 'collection',
+            'moduleName' => 'artworks',
+        ],
+
+        // Exhibition and events landing
+        'exhibitionsExhibitions' => [
+            'moduleName' => 'exhibitions',
+            'routePrefix' => 'exhibitions_events'
+        ],
+        'exhibitionsCurrent' => [
+            'moduleName' => 'exhibitions',
+            'routePrefix' => 'exhibitions_events'
+        ],
+        'exhibitionsUpcoming' => [
+            'moduleName' => 'exhibitions',
+            'routePrefix' => 'exhibitions_events'
+        ],
+        'exhibitionsUpcomingListing' => [
+            'moduleName' => 'exhibitions',
+            'routePrefix' => 'exhibitions_events'
+        ],
+
+        // Collection landing
+        'artCategoryTerms' => [
+            'moduleName' => 'categoryTerms',
+            'routePrefix' => 'collection',
+        ],
+
+    ];
+
+    protected $repeaters = [
+        // Homepage landing
+        'artists' => [
+            'relation' => 'homeArtists',
+            'model' => 'HomeArtist'
+        ],
+
+        // Visit
+        'locations',
+        'featured_hours' => [
+            'relation' => 'featured_hours'
+        ],
+        'faqs',
+        'families',
+        'what_to_expects',
+    ];
+
+    public function __construct(LandingPage $model)
+    {
+        $this->model = $model;
+    }
+
+    public function hydrate($object, $fields)
+    {
+        $this->hydrateOrderedBelongsTomany($object, $fields, 'homeExhibitions', 'position', 'Exhibition');
+        $this->hydrateOrderedBelongsTomany($object, $fields, 'homeEvents', 'position', 'Event');
+        $this->hydrateOrderedBelongsTomany($object, $fields, 'homeFeatures', 'position', 'HomeFeature');
+        $this->hydrateOrderedBelongsTomany($object, $fields, 'mainHomeFeatures', 'position', 'HomeFeature');
+        $this->hydrateOrderedBelongsTomany($object, $fields, 'secondaryHomeFeatures', 'position', 'HomeFeature');
+        $this->hydrateOrderedBelongsTomany($object, $fields, 'homeShopItems', 'position', 'ShopItem');
+        $this->hydrateOrderedBelongsTomany($object, $fields, 'visitTourPages', 'position', 'GenericPage');
+        $this->hydrateOrderedBelongsTomany($object, $fields, 'researchResourcesFeaturePages', 'position', 'GenericPage');
+
+        $this->hydrateOrderedBelongsTomany($object, $fields, 'researchResourcesStudyRooms', 'position', 'GenericPage');
+        $this->hydrateOrderedBelongsTomany($object, $fields, 'researchResourcesStudyRoomMore', 'position', 'GenericPage');
+
+        $this->hydrateOrderedBelongsTomany($object, $fields, 'articles', 'position', 'Article');
+        $this->hydrateOrderedBelongsTomany($object, $fields, 'printedPublications', 'position', 'PrintedPublication');
+        $this->hydrateOrderedBelongsTomany($object, $fields, 'digitalPublications', 'position', 'DigitalPublication');
+
+        $this->hydrateOrderedBelongsTomany($object, $fields, 'homeArtists', 'position', 'HomeArtist');
```

Comment:
The 'm' in 'many' needs to be capitalized, see https://github.com/area17/twill/blob/3.x/src/Repositories/Behaviors/HandleRevisions.php#L123

Reply:
Still need capitalization